src/tests/ftest/rebuild/continues_after_stop.py

```diff
@@ -0,0 +1,125 @@
+        """Run IOR once with configurations in the test yaml.
+
+        Args:
+            namespace (str): Namespace that defines block_size and transfer_size.
+            pool (TestPool): Pool to use with IOR.
+            container (TestContainer): Container to use with IOR.
+        """
+        ior_cmd = IorCommand(namespace=namespace)
+        ior_cmd.get_params(self)
+        ior_cmd.set_daos_params(self.server_group, pool, container.identifier)
+        testfile = os.path.join(os.sep, "test_file_1")
+        ior_cmd.test_file.update(testfile)
+        manager = get_job_manager(test=self, job=ior_cmd, subprocess=self.subprocess)
+        manager.assign_hosts(
+            self.hostlist_clients, self.workdir, self.hostfile_clients_slots)
+        ppn = self.params.get("ppn", namespace)
+        manager.assign_processes(ppn=ppn)
+
+        try:
+            manager.run()
+        except CommandFailure as error:
+            self.log.info(error)
+
+    def test_continuous_after_stop(self):
+        """Verify rebuild continues after one of the ranks is stopped.
+
+        1. Create a pool and a container.
+        2. Run IOR that takes several seconds with a thread.
+        3. After a few seconds, stop one of the ranks (rank 3).
+        4. Look for the start of the rebuild (Rebuild [scanning]) in journalctl with daos_server
+        identifier.
+        5. As soon as the message is detected, stop the rest of the ranks (0, 1, 2).
+        6. Restart the three ranks.
+        7. Wait for rebuild to finish.
+
+        Jira ID: DAOS-6287
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=rebuild
+        :avocado: tags=ContinuesAfterStop,test_continuous_after_stop
+        """
+        # 1. Create a pool and a container.
+        self.log_step("Create a pool and a container.")
+        pool = self.get_pool()
+        container = self.get_container(pool=pool)
+
+        # 2. Run IOR that takes several seconds with a thread.
+        self.log_step("Run IOR that takes several seconds with a thread.")
+        kwargs = {
+            "namespace": "/run/ior/*",
+            "pool": pool,
+            "container": container
+        }
+        thread = threading.Thread(target=self.run_ior_basic, kwargs=kwargs)
+        ior_start_time = journalctl_time()
+        thread.start()
+
+        # 3. After a few seconds, stop one of the ranks (rank 3).
+        self.log_step("After a few seconds, stop one of the ranks (rank 3).")
+        # Wait for IOR to start and write some data. Otherwise rebuild will not occur.
+        time.sleep(5)
+        self.server_managers[0].stop_ranks(ranks=[3], daos_log=self.log)
+
+        # 4. Look for the start of the rebuild (Rebuild [scanning]) in journalctl with daos_server
+        # identifier.
+        msg = ("4. Look for the start of the rebuild (Rebuild [scanning]) in journalctl with "
+               "daos_server identifier.")
```

Comment:
I think we could just remove the duplicate comments in favor of the log_step message

Reply:
Done